packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -8067,6 +8067,25 @@ void main() {
     );
   });
 
+  testWidgets('Cupertino text field control suffix and prefix widgets alignment', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        const CupertinoApp(
+          home: CupertinoTextField(
+            prefix: Icon(CupertinoIcons.add),
+            suffix: Icon(CupertinoIcons.clear),
+            crossAxisAlignment: CrossAxisAlignment.start,
+          ),
+        ),
+      );
+
+      final CupertinoTextField cupertinoTextField = tester.widget<CupertinoTextField>(
+        find.byType(CupertinoTextField),
+      );
+
+      expect(cupertinoTextField.crossAxisAlignment, CrossAxisAlignment.start);
```

Comment:
It's not enough just to show that the parameter was received, could you also test that it's having the intended effect? Maybe find the prefix/suffix and check its position.